src/controllers/novelController.js

```diff
@@ -0,0 +1,28 @@
+import createNewNovel from '../services/createNovel';
+import helpers from '../helpers';
+
+
+const { errorResponse, successResponse } = helpers;
+
+/**
+ * createNovel
+ *
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} json
+ */
+const createNovel = async (req, res) => {
+  const newNovel = req.body;
```

Comment:
what happens if a column that's not in the database is sent in the request body?

Reply:
I wrote validations to prevent this from happening using the validation middleware